webp/view/src/main/cpp/webp_decode.cpp

```diff
@@ -111,6 +112,7 @@ void WebpDecoder::DecodeFrameInternal() {
             break;
         default:
             assert( 0 );
+            delete  [] buf;
```

Comment:
Okay. This should probably all be cleaned up at some point, but it doesn't need to block this change. When it does get cleaned up, only use asserts for things where you do want to continue and explicitly abort when you can't.